apps/pwabuilder-chromium-extension/package.json

```diff
@@ -6,7 +6,10 @@
   "scripts": {
     "dev": "webpack --config webpack.config.js --watch",
     "preinstall": "node ../../scripts/setupDeps.js",
-    "build": "webpack --config webpack.config.js"
+    "build": "webpack --config webpack.config.js",
+    "build:components": "webpack --config webpack.config.components.js --watch",
```

Comment:
I'm not sure the --watch belongs on a build command, doesn't that result in it continuing to run?

Reply:
done